test/cpp/end2end/xds/xds_end2end_test.cc

```diff
@@ -10846,93 +10757,6 @@ TEST_P(XdsRbacTestWithActionPermutations, AnyPermissionMethodPostPrincipal) {
   SendRpc([this]() { return CreateInsecureChannel(); }, {}, {},
           /*test_expects_failure=*/GetParam().rbac_action() == RBAC_Action_DENY,
           grpc::StatusCode::PERMISSION_DENIED);
-  // Test an RPC with a different method type
```

Comment:
@yashykt Same question here.

Reply:
Same here. I guess we'll need testing (or some way to be confident) that all other method types are rejected. Otherwise, this test is not really needed.

Reply:
Same as above: I think we still need tests here for both the positive and negative match cases.